Library/Homebrew/utils/fork.rb

```diff
@@ -2,6 +2,26 @@
 require "socket"
 
 module Utils
+  def self.rewrite_child_error(child_error)
+    error = if child_error.inner_class == ErrorDuringExecution
```

Comment:
Maybe a `case` here?

Reply:
`case` uses `#===` for comparison, so comparing a class directly doesn't work :disappointed::

```ruby
case String
when String then :foo
else :bar
end # => :bar
```